make-webpack-config.js

```diff
@@ -0,0 +1,172 @@
+var path = require('path');
+var fs = require('fs');
+var webpack = require('webpack');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var StatsPlugin = require('stats-webpack-plugin');
+var loadersByExtension = require('./webpack/lib/loadersByExtension');
+
+module.exports = function(options) {
+    var VERSION = require('./package.json').version;
+    var PACKAGE_NAME = require('./package.json').name;
+    var LICENSE = fs.readFileSync('LICENSE', 'utf8');
+
+    var entry = {
+        smooch: './src/js/main'
+    };
+
+    var loaders = {
+        'jsx': options.hotComponents ? ['react-hot-loader', 'babel-loader'] : 'babel-loader',
+        'js': {
+            loader: 'babel-loader',
+            include: path.join(__dirname, 'src/js')
+        },
+        'json': 'json-loader',
```

Comment:
oh, yeah since the config is required somewhere in the code, the bundle will rebuild itself if changed but it won't trigger a hot-module replacement.. maybe a full page reload though